nsxt/resource_nsxt_policy_shared_resource.go

```diff
@@ -0,0 +1,207 @@
+/* Copyright © 2024 Broadcom, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/terraform-provider-nsxt/api/infra/shares"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+)
+
+func resourceNsxtPolicySharedResource() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicySharedResourceCreate,
+		Read:   resourceNsxtPolicySharedResourceRead,
+		Update: resourceNsxtPolicySharedResourceUpdate,
+		Delete: resourceNsxtPolicySharedResourceDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceNsxtPolicySharedResourceImporter,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
I see a minor but non-null value in adding `sharing-shared-with` as a computed attribute. What do you reckon?

Reply:
Yeah it could be useful. I'll add ir.

Reply:
So I've tried to add this - but NSX returns nothing in this attr. I'll remove it then.